src/main/java/hudson/plugins/templateproject/ProxySCM.java

```diff
@@ -82,7 +78,7 @@ public void checkout(@Nonnull Run<?,?> build, @Nonnull Launcher launcher, @Nonnu
 		// https://github.com/jenkinsci/multiple-scms-plugin/pull/6
 		// https://issues.jenkins-ci.org/browse/JENKINS-27638
 		// Since MultiSCM is optional, might not be installed, hacky check string name.
-		if (getProjectScm((AbstractBuild) build).toString().contains("multiplescms")) {
+		if(Jenkins.getInstance() != null && Jenkins.getInstance().getPlugin("multiple-scms") != null) {
```

Comment:
Could you use ```Jenkins.get()```, because ```getInstance()``` is deprecated. Don't need to test for null

Reply:
It doesn't exists in the minimum core requirement of this plugin. I would if I could.

Reply:
not 100% sure, but I think he can not do it. The parent pom is too old, so that method probably is not available for this plugin. @alecharp, have you consider upgrade the parent pom to a newer version?

Reply:
I prefer not upgrading the parent pom nor Jenkins Core version requirement if it's not necessary for the PR. Limiting the impact of the changeset.

Reply:
Fair enough